roles/v1.24/kiali-deploy/defaults/main.yml

```
@@ -167,6 +167,9 @@ kiali_defaults:
       namespace_selector: true
       url: ""
       whitelist_istio_system: ["jaeger-query", "istio-ingressgateway"]
+      
+  health_config:
+    rate: [ ]
```

Please make sure all of these files across branches/versions match as close as possible. It helps immensely when we need to do a "diff" across all these different locations to find out what really changes between branches and versions. So delete the space in between the brackets to match the default branch as you see here: https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/defaults/main.yml#L176
```suggestion
    rate: []
```